reference/docs-conceptual/Crescendo/get-started/choose-command-line-tool.md

```diff
@@ -0,0 +1,73 @@
+---
+description: How to select a command-line tool to amplify using Crescendo.
+ms.date: 03/08/2022
+title: Choosing a command-line tool to amplify
+---
+# Choosing the command-line tool for Crescendo
+
+Developing a cmdlet with Crescendo is a rapid, reliable way to _amplify_ a command-line tool to
+produce a cmdlet-like experience. Many times, the tool you need to work with is directly tied to the
```

Comment:
~~Been thinking about this a little bit during the broader review; I think it is worth explicitly calling out that the end product is a cmdlet _but_ "developing a cmdlet... to produce a cmdlet-like experience" feels a little off. This also introduces the "developing a cmdlet" versus "amplifying a command-line tool" terminology distinction I mention elsewhere. I keep trying to rephrase this line without losing the point of it (from my perspective, my key take away as a reader is "Crescendo helps me rapidly and reliably _amplify_ a command-line tool to a cmdlet-like experience"). My best effort:~~
```suggestion
Using Crescendo is a rapid, reliable way to _amplify_ a command-line tool to
produce a cmdlet-like experience. Many times, the tool you need to work with is directly tied to the
```

Edit: Stricken in favor of [this comment]() but left for clarity (and in case the decision is made to update terminology).